README.md

```diff
@@ -69,7 +69,13 @@ end
 3) Use the view helper to do all the process:
 
 ```ruby
-<%= nexaas_async_collect(user.id, ModelService, :model_method, [arg1, arg2]) %>
+<%= nexaas_async_collect({
+  user_id: user.id, # (required) the ID of the user. It ensures only the user who requested the data will be able to fetch it
```

Comment:
I don't know if `user_id` is the best name since we can use an scope that is not "user". Like we say at readme:

```
# The method that returns the user object (or any other object you want. It must respond to id method)
config.scope = :current_user
```